protocol/app/upgrades/v5.0.0/constants.go

```diff
@@ -18,6 +19,7 @@ var Upgrade = upgrades.Upgrade{
 	StoreUpgrades: store.StoreUpgrades{
 		Added: []string{
 			vaulttypes.StoreKey,
+			wasmtypes.StoreKey,
```

Comment:
This should not be in v5.0.0 upgrade which has already happened. We don't yet know which version will CosmWasm be released in, could you create a dummy upgrade version (`v_upgrade_cosmwasm`)?

Reply:
I was going to put it in v6 after merging with main. I agree this is a better approach though

Reply:
Actually, for now - it may be better to just skip upgrade tests(let container tests fail) than adding the extra code since we arent doing anything for container tests yet. I can add this in later when we are closer to merging

Reply:
changed my mind, added upgrade tests but duplicated a bit of code. Will remove duplicated code on actual upgrade

Reply:
Again, we should not be modifying anything related to `v5.0.0` upgrade. This needs to be in `v.cosmawm.0`

Reply:
Missed this. Good catch!